controllers/imageclusterinstall_controller.go

```diff
@@ -384,6 +386,9 @@ func (r *ImageClusterInstallReconciler) writeInputData(ctx context.Context, log
 
 	locked, lockErr, funcErr := filelock.WithWriteLock(lockDir, func() error {
 
+		if err := r.writeInvokerCM(filepath.Join(clusterConfigPath, invokerCMFileName)); err != nil {
+			return fmt.Errorf("failed to write ca bundle: %w", err)
```

Comment:
I think it is copy paste text issue.
Shouldn't it be something like 
"failed to create invoker configmap" ?